packages/bpk-component-info-banner/src/BpkInfoBanner.module.scss

```diff
@@ -0,0 +1,115 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '../../bpk-mixins/index.scss';
+
+.bpk-info-banner {
+  padding: bpk-spacing-md() bpk-spacing-base();
+  overflow: hidden;
+
+  @include info-banner;
+
+  &--style-default {
+    background-color: $bpk-canvas-contrast-day;
+  }
+
+  &--style-onContrast {
+    background-color: $bpk-surface-default-day;
+  }
+
+  &__header {
+    display: flex;
+
+    // padding: bpk-spacing-md() bpk-spacing-base();
+    flex-direction: row;
+    align-items: flex-start;
+
+    &--expandable {
+      cursor: pointer;
+    }
+  }
+
+  &__icon,
+  &__toggle {
+    padding: bpk-spacing-sm() / 2 0;
+  }
+
+  // Make sure the items in the flexbox are display: inline-block; for older IE's
```

Comment:
We don't support IE anymore so is this a concern?

Reply:
Flexbox seems to be [working](https://caniuse.com/flexbox) in all modern browsers. And I don't see any different behaviour when I delete it. Though, this `inline-block` display is specified as one of the [fallback techniques](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_flexible_box_layout/Backwards_compatibility_of_flexbox#display_inline-block) for flexbox.
Do we want to delete it, or it should remain just in case there's some older browsers that some users might have? According to the table in first link, it's a very rare occasions.

Reply:
I checked with the current browser support level and we don't actually support any of the older browsers and the levels of support mentioned in the first link covers what our product supports so I think here we are safe to remove it